packages/optimizer/README.md

```diff
@@ -48,17 +50,214 @@ ampOptimizer.transformHtml(originalHtml).then((optimizedHtml) => {
 
 You can find a sample implementation [here](/packages/optimizer/demo/simple). If you're using express to serve your site, you can use the [AMP Optimizer Middleware](/packages/optimizer-express).
 
-### Incomplete markup
+### CLI
+
+AMP Optimizer can be used via the [AMP Toolbox CLI](/packages/cli/README.md):
+
+```shell
+$ npm install @ampproject/toolbox-cli -g
+$ amp optimize myFile.html
+```
+
+or run without installation via `npx`:
+
+```shell
+$ npx @ampproject/toolbox-cli optimize myFile.html
+```
+
+### Options
+
+Options are passed when creating a new AMP Optimizer instance:
+
+```
+const ampOptimizer = AmpOptimizer.create({
+  verbose: true
+});
+...
+```
+
+Available options are:
+
+* [autoAddBoilerplate](#autoaddboilerplate)
+* [autoExtensionImport](#autoextensionimport)
+* [format](#format)
+* [imageBasePath](#imagebasePath)
+* [imageOptimizer](#imageoptimizer)
+* [lts](#lts)
+* [markdown](#markdown)
+* [minify](#minify)
+* [preloadHeroImage](#preloadheroimage)
+* [verbose](#verbose)
+
+#### `autoAddBoilerplate`
 
-It's possible to pass incomplete documents and AMP Optimizer will add any 
-missing tags and extension imports required by a valid AMP document. 
+Automatically inject any missing markup required by AMP.
+
+* name: `autoAddBoilerplate`
```

Comment:
this isn't AMP Boilerplate, right? naming is confusing if I am correct (since you advocate for it being removed laster on)

Reply:
Yup, but changing this would be a breaking API change :-(. Anyway, renamed it to autoAddMandatoryTags and kept the old name as well.